lib/streamlit/server/server.py

```diff
@@ -700,22 +697,23 @@ def initialize(self, server):
         # See https://www.tornadoweb.org/en/stable/guide/security.html#cross-site-request-forgery-protection
         # for more details.
         if config.get_option("server.enableXsrfProtection"):
-            self.xsrf_token
+            _ = self.xsrf_token
```

Comment:
what does this line do?

Reply:
This one is explained by the comment above the line

Reply:
my bad, the diff only showed the url